changelogs/unreleased/improve-typing-performance.yml

```diff
@@ -0,0 +1,5 @@
+description: Improve performance of duplicate detection in compiler typing
+change-type: patch
```

Comment:
If this is a patch this should also go into iso4 no?

Reply:
It is an improvement, so I would not make the effort to put it in iso4.

iso4 should only receive the things that it really needs.

Reply:
Then this should be made a minor change? My point was that mergetool will (I think) reject the PR as it is now